rust/metaplex/program/src/processor/empty_payment_account.rs

```diff
@@ -85,11 +85,10 @@ fn calculate_owed_amount(
     winning_config_item_index: &Option<u8>,
     creator_index: &Option<u8>,
 ) -> Result<u64, ProgramError> {
-    let primary_sale_happened = auction_manager.get_primary_sale_happened(
-        metadata,
-        *winning_config_index,
-        *winning_config_item_index,
-    )?;
+    let should_not_split_as_primary_sale = match auction_manager.get_auction_manager_status() {
+        Ok(AuctionManagerStatus::Disbursing) => false,
```

Comment:
I'm not getting how state of disbursing should disable the royalty split.

Reply:
It's not super clear at a glance but the auction manager is set to disbursing before closing after the nft have been claimed.

Reply:
Auction is used to resale already owned NFTs as well so this would kick in if the NFT was sold already.

Reply:
Empty payment account is already passing in the safety deposit box. Think this is our new spot to snapshot the sale status and read for primary sale.